libraries/lib-effects/MixAndRender.cpp

```diff
@@ -19,17 +19,15 @@ Paul Licameli split from Mix.cpp
 using WaveTrackConstArray = std::vector < std::shared_ptr < const WaveTrack > >;
 
 //TODO-MB: wouldn't it make more sense to DELETE the time track after 'mix and render'?
-void MixAndRender(const TrackIterRange<const WaveTrack> &trackRange,
+TrackListHolder MixAndRender(const TrackIterRange<const WaveTrack> &trackRange,
```

Comment:
I vote this for the best commit of this PR :-)

Reply:
Curious, why this one in particular?